tests/bluetooth/tester/src/mesh.c

```diff
@@ -13,6 +13,11 @@
 #include <misc/byteorder.h>
 #include "bttester.h"
 
+#include "../../../../subsys/bluetooth/host/mesh/foundation.h"
+#include "../../../../subsys/bluetooth/host/mesh/mesh.h"
+#include "../../../../subsys/bluetooth/host/mesh/net.h"
+#include "../../../../subsys/bluetooth/host/mesh/transport.h"
```

Comment:
Did you consider having the tester register a dummy client model and do the sending of messages using public APIs as if the messages belong to that model? That'd let us avoid having to include private headers.

Reply:
you mean to send via bt_mesh_model_send? I will try.

Reply:
yes, that's what I meant

Reply:
note that you'll probably want to use a vendor model, e.g. with CID 0xffff

Reply:
and I just realized you even have a define for that already: CID_LOCAL

Reply:
Yes, I added this for composition data.

Reply:
@jhedberg The problem is I don't have app_key. Should I add application key, just like you did in mesh_demo? Using BT_MESH_KEY_DEV failed for me with error:
"model_send: Model not bound to AppKey 0xfffe".
I tried also BT_MESH_KEY_UNUSED, but then bt_mesh_app_key_find() returns NULL, and send fails with -EINVAL error.